File: SaasFeeGuides/SaasFeeGuides.ViewModels/ActivitySku.cs
using System.Collections.Generic;

namespace SaasFeeGuides.ViewModels
{
    public class ActivitySku
    {
        public string ActivityName { get; set; }
        public string Name { get; set; }
        public IList<Content> TitleContent { get; set; }
        public IList<Content> DescriptionContent { get; set; }
        public double PricePerPerson { get;  set; }
        public int MinPersons { get; set; }
        public int MaxPersons { get; set; }
        public IList<Content> AdditionalRequirementsContent { get; set; }
        public double DurationDays { get;  set; }
        public double DurationHours { get; set; }
        public IList<Content> WebContent { get; set; }
        public int ActivityId { get; set; }
        public int Id { get; set; }
    }
}

File: SaasFeeGuides/SaasFeeGuides/ClientApp/src/app/viewModels/activitySku.ts

import { Content } from './content';
import { ActivitySkuPrice } from './activitySkuPrice';


export class ActivitySku {

  constructor(
    public id: number,
    public activityName: string,
    public name: string,
    public titleContent: Content[],
    public descriptionContent: Content[],
    public pricePerPerson: number,
    public minPersons: number,
    public maxPersons: number,
    public additionalRequirementsContent: Content[],
    public durationDays: number,
    public durationHours: number,
    public webContentId: string,
    public activityId: number,
    public priceOptions: ActivitySkuPrice[]

  ) {
  }

}


